### PR TITLE
Page Templates: karma keyboard tests

### LINKED ISSUE #6109: Page Templates: Add Keyboard karma tests
## Feature Description

Add karma tests that ensure keyboard usability of page layouts. 

## Additional Context

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

### PR DESCRIPTION
**Pull request by BrookeGraham**
## Summary
Add karma tests that ensure keyboard usability of page layouts.

## User-facing changes

n/a

## Testing Instructions

- [x] This is a non-user-facing change and requires no QA

This PR can be tested by following these steps:

1.

### Does this PR have a security-related impact?
no

### Does this PR change what data or activity we track or use?
no

### Does this PR have a legal-related impact?
no

## Checklist

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [x] This code is covered by automated tests (unit, integration, and/or e2e) to verify it works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [x] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #6109

### CODE REVIEW

File: packages/story-editor/src/dataUtils/formattedTemplatesArray.js [hunk 1]
```diff
@@ -1268,8 +1268,12 @@ const formattedTemplatesArray = [
     ],
     postersByPage: [
       { png: 'http://placekitten.com/158/237', type: 'cover' },
-      { png: 'http://placekitten.com/158/237', type: 'list' },
+      { png: 'http://placekitten.com/158/237', type: 'section' },
       { png: 'http://placekitten.com/158/237', type: 'quote' },
+      { png: 'http://placekitten.com/158/237', type: 'editorial' },
+      { png: 'http://placekitten.com/158/237', type: 'list' },
+      { png: 'http://placekitten.com/158/237', type: 'table' },
+      { png: 'http://placekitten.com/158/237', type: 'steps' },
```
Comment (BrookeGraham) [inline packages/story-editor/src/dataUtils/formattedTemplatesArray.js:1276]: I updated these to match the `pages` so that my filtering tests would work.

**Comment by BrookeGraham** [comment]
Should I add keyboard tests for saved templates as well? @BrittanyIRL

**Review by samwhale — APPROVED** [review]
Noice, tests work well and match expected behavior on my machine.

**Review by maxyinger — APPROVED** [review]
nice. lgtm

**Comment by BrittanyIRL** [comment]
> Should I add keyboard tests for saved templates as well? @BrittanyIRL

nah! we can do that with further updates to the custom page templates

**Review by BrittanyIRL — APPROVED** [review]
LGTM!

**Pull request merged**